app/views/play/SpectateView.coffee

```diff
@@ -106,6 +106,13 @@ module.exports = class SpectateLevelView extends RootView
       spectateMode: true
       team: utils.getQueryVariable('team')
     @god = new God maxAngels: 1, spectate: true
+    @anonymousPlayerName = false
+    if (league = utils.getQueryVariable('league')) and (@anonymousPlayerName = features.enableAnonymization)
+      fetchAnonymous = $.get('/esports/anonymous/' + league)
```

Comment:
this calling API and setting `@anonymousePlayerName` repeats few times. Can you refactor it?

Reply:
yeah, i would refactor to utils.